packages/components/src/ListBox/docs/migration-notes.md

```diff
@@ -1,5 +1,5 @@
-- `disabled` renamed to `isDisabled`.
-- `validationState` is removed. use `isInvalid` instead. There is no `isValid`.
-- `tabbable` is removed.
+- `disabled` has been renamed to `isDisabled`.
+- `validationState` has been removed, use `isInvalid` instead. `valid` has been removed.
```

Comment:
```suggestion
- `validationState` has been removed. `isInvalid` should be used instead.
```
That exactly right to say that `valid` has been removed since that refers to a value of validationState.
The reason why we said "There is no `isValid`" was because we added isInvalid, but no isValid to replace the valid part of validationState. We are not doing that style. We're only doing an invalid style.